src/main/java/org/jenkinsci/plugins/github/extension/GHEventsSubscriber.java

```diff
@@ -23,7 +22,7 @@
 
 /**
  * Extension point to subscribe events from GH, which plugin interested in.
- * This point should return true in {@link #isApplicable(AbstractProject)}
+ * This point should return true in {@link #isApplicable}
```

Comment:
I beleive this is an invalid link would refer to the non-existant field `isApplicable` unless something has changed?